lib/tweets.rb

```diff
@@ -0,0 +1,67 @@
+require 'pg'
+
+class Tweets
+
+  attr_reader :id, :message, :user_id
+
+  def initialize(id:, message:, user_id:)
+    @id = id
+    @message = message
+    @user_id = user_id 
+  end
+
+def self.all
+  if ENV['ENVIRONMENT'] == 'test'
```

Comment:
It's worth considering that this if/else statement is repeated below as well, to follow the DRY principle, you could, for example, create a function that takes the `environment` information and then reuse it to establish the connection with the test or the main database.